services/src/main/java/org/keycloak/protocol/oidc/grants/PreAuthorizedCodeGrantTypeFactory.java

```diff
@@ -0,0 +1,38 @@
+package org.keycloak.protocol.oidc.grants;
```

Comment:
Could you add header comments the same as other newly created files?

Reply:
[7beca5e](https://github.com/keycloak/keycloak/pull/27931/commits/7beca5e54dc584c6726a73ca82109f0fc3d22124)

Reply:
Thank you.